pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -194,6 +206,14 @@ def __init__(
                 ' Please prepend your prefix in `filename` instead.', DeprecationWarning
             )
 
+        if period is not None:
+            rank_zero_warn(
+                'Argument `period` is deprecated in v1.3 and will be removed in v1.5.'
+                ' Please use `every_n_epochs` instead.', DeprecationWarning
+            )
+            self.every_n_epochs = period
+            self.period = period
```

Comment:
this needs proper deprecation tests for 1.5 - please advise on best practices for it